test/SqlExtension.Tests/SqlInputBindingIntegrationTests.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SqlExtensionSamples;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SqlExtension.Tests
```

Comment:
It would be convenient to put all the tests into separate folders and under a different namespace based on their type (`SqlExtension.Tests.Integration` and `SqlExtension.Tests.Unit` for example)